portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsParagraphField.testcase

```diff
@@ -277,8 +277,56 @@ definition {
 		property forms.functional = "false";
 		property portal.acceptance = "false";
 
-		// TODO LPS-131237 CanDeleteATableUsingBackspace pending implementation
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName(formName = "Form 1");
+
+		Form.gotoAddField(
+			fieldPositionNumber = "0",
+			fieldType = "Paragraph");
+
+		Click.clickNoMouseOver(
+			key_titleName = "Table",
+			locator1 = "FormFields#TOOLBAR_ANY_ITEM_BUTTON");
+
+		Click(locator1 = "CKEditor#OK_BUTTON");
+
+		Form.save();
+
+		Click(
+			key_fieldName = "Paragraph",
+			locator1 = "FormFields#PARAGRAPH_BODY_TEXT_CONTENT");
+
+		FormFields.deleteTextWithBackspace();
+
+		FormFields.viewTableNotPresent();
+
+		Type.clickAtType(
+			locator1 = "CKEditor#BODY",
+			value1 = "Test");
+
+		SelectFrame(value1 = "relative=top");
+
+		Form.save();
+
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+		Click(
+			key_formName = "Form 1",
+			locator1 = "FormsAdmin#FORMS_TABLE_NAME");
```

Comment:
Hi @leilanyaragao , you can use the macro FormsAdminNavigator.gotoEditFormByName() to go back to Form 1.